.semgrep/rpc_endpoint.yml

```diff
@@ -89,6 +89,7 @@ rules:
           patterns:
             # Endpoints that are expected not to have authentication.
             - pattern-not: '"ACL.Bootstrap"'
+            - pattern-not: '"ACL.GetClaimPolicies"'
```

Comment:
Could a semgrep expert quick peek at these patterns and my RPC and make sure this is the right choice?

The intent of `ACL.GetClaimPolicies` is to do a lookup using the authenticated claim which is a fairly new pattern, so I think it just escapes semgreps matching.

Reply:
Yeah that's right, it's similar to the `ACL.ResolveToken` case in that way. Arguably the name of this rule `rpc-potentially-unauthenticated` should be changed... once authentication and authorization got split out this is really testing for authorization.

Reply:
> once authentication and authorization got split out this is really testing for authorization.

Ah thanks for making this distinction explicit! It's quite the mental shift for me after years of not having to think about them independently.

I updated a comment but am too nervous to make a more invasive change in this PR. :sweat_smile: